esx_service/utils/auth_data.py

```diff
@@ -734,24 +734,29 @@ def err_config_init_needed(self):
         return error_code_to_message[ErrorCode.INIT_NEEDED]
 
 
-    def new_db(self):
+    def __init_db(self):
         """
         Create brand new DB content at self.db_path. Expects clean slate.
         :returns: None for success , a string (with error) for error
         """
         if not self.conn:
             self.__connect()
         err = self.__create_tables()
+        if not err:
```

Comment:
Nit: I feel the previous style is simple and straightforward. If we really want to avoid these trivial if-check, we should have a nested structure like this:
    if not err:
        err = xxxx
            if not err:
                err = xxxx
                if not err:
                    err = xxxx
I think this is an overkill. So I'd prefer to keep the previous style.